lftp/ftpserver.py

```diff
@@ -51,7 +51,7 @@ def setup_ftp(self):
         handler.abstracted_fs = UnifiedFilesystem
         handler.abstracted_fs.basepaths = basepaths
         authorizer.add_anonymous(basepaths[0])
-        address = (self.config['ftp.host'], self.config['ftp.port'])
+        address = ('', self.config['ftp.port'])
```

Comment:
It does not. When given a blank address, it opens up the port on all interfaces.

The config key for librarian is `app.bind` ?